internal/action/generate.go

```diff
@@ -231,6 +231,7 @@ func (s *Action) generatePasswordForRule(ctx context.Context, c *cli.Context, le
 	if err != nil {
```

Comment:
Shouldn't we reject invalid lengths instead of just clamping them below?

Reply:
Maybe. But either we make the implementation much more convoluted (loops etc.) or we abort the execution and possibly disrupt the user. Not sure either. But not clamping or checking them also sounds wrong.